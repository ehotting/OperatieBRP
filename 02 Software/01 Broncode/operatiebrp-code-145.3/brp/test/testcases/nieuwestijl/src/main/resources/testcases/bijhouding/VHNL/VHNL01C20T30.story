Meta:
@auteur                 tjlee
@status                 Klaar
@sleutelwoorden         Geslaagd
@regels                 R1572
@usecase                UCS-BY.HG

Narrative:
R1572 Huwelijk en geregistreerd partnerschap, voltrekking huwelijk in Nederland, actie registratieAanvangHuwelijkGeregistreerdPartnerschap

Scenario: R1572 Geen overlapping Ouder.Ouderschap.Datum geldigheid met Geslachtsnaamcomponent.Standaard.Datum geldigheid
          LT: VHNL01C20T30



Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL/VHNL01C20T30-sandy.xls
Given enkel initiele vulling uit bestand /LO3PL/VHNL01C20T30-danny.xls

When voer een bijhouding uit VHNL01C20T30.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Geslaagd
And is het antwoordbericht gelijk aan /testcases/bijhouding/VHNL/expected/VHNL01C20T30.xml voor expressie //brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R

Then is in de database de persoon met bsn 558376617 wel als PARTNER betrokken bij een HUWELIJK
Then is in de database de persoon met bsn 156960849 wel als PARTNER betrokken bij een HUWELIJK

Then lees persoon met anummer 1686982546 uit database en vergelijk met expected VHNL01C20T30-persoon1.xml
Then lees persoon met anummer 3814796818 uit database en vergelijk met expected VHNL01C20T30-persoon2.xml
