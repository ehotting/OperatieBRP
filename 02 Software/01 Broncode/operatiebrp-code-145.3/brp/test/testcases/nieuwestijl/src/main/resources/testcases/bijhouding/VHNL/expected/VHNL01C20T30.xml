<?xml version="1.0" encoding="UTF-8"?><brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
   <brp:stuurgegevens>
      <brp:zendendePartij>199901</brp:zendendePartij>
      <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
      <brp:referentienummer>*</brp:referentienummer>
      <brp:crossReferentienummer>00000000-0000-0000-0042-200000000104</brp:crossReferentienummer>
      <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
   </brp:stuurgegevens>
   <brp:resultaat>
      <brp:verwerking>Geslaagd</brp:verwerking>
      <brp:bijhouding>Verwerkt</brp:bijhouding>
      <brp:hoogsteMeldingsniveau>Waarschuwing</brp:hoogsteMeldingsniveau>
   </brp:resultaat>
   <brp:meldingen>
        <brp:melding brp:objecttype="Melding" brp:referentieID="comid.voltrekkingHuwelijkInNederland.registratieGeslachtsnaam.persoon">
            <brp:regelCode>R1572</brp:regelCode>
            <brp:soortNaam>Waarschuwing</brp:soortNaam>
            <brp:melding>*</brp:melding>
        </brp:melding>
    </brp:meldingen>
   <brp:voltrekkingHuwelijkInNederland brp:objecttype="AdministratieveHandeling">
      <brp:partijCode>507013</brp:partijCode>
      <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>
       <brp:bijgehoudenPersonen>*</brp:bijgehoudenPersonen>
        <brp:bijhoudingsplan brp:objecttype="Bijhoudingsplan">*</brp:bijhoudingsplan>
   </brp:voltrekkingHuwelijkInNederland>
</brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R>
